File: 6/stringreview.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <iostream>
#include <iomanip>
#include <limits>

using namespace std;


int main() {
    int N;
    cin >> N;
    cin.ignore();

    string line;
    string first = "";
    string second = "";

	for (int i = 0; i < N; i++) {
      getline(cin, line);
      first = "";
      second = "";
      for (int c = 0; c < line.length(); c++) {
        if (c % 2 == 0) {
          first += line[c];
        } else {
          second += line[c];
        }
      }
      cout << first << " " << second << endl;
	}

    return 0;
}

File: 6/stringreview.js
process.stdin.resume();
process.stdin.setEncoding('ascii');

var input_stdin = "";
var input_stdin_array = "";
var input_currentline = 0;

process.stdin.on('data', function (data) {
    input_stdin += data;
});

process.stdin.on('end', function () {
    input_stdin_array = input_stdin.split("\n");
    main();    
});

function readLine() {
    return input_stdin_array[input_currentline++];
}

/////////////// ignore above this line ////////////////////

function walk(line) {
  var first = [];
  var second = [];
  for (i = 0; i < line.length; i++) {
    if (i % 2 == 0) {
      first.push(line[i]);
    } else {
      second.push(line[i]);
    }
  }
  var out = first.join("") + " " + second.join("");
  return out;
}

function main() {
  var N = parseInt(readLine());
  var statement = "";
  for (var i = 1; i <= N; i++) {

    var line = readLine();
    statement = walk(line) + "\n";
    process.stdout.write(statement);
  }
}

